### PR TITLE
Fix double-quoted warnings in Xcode 12

### PR DESCRIPTION
**Pull request by AnastasiaKubova**
Things to consider before you submit the PR:

* [x] Has `CHANGELOG.md` been updated?
* [x] Are tests passing locally?
* [x] Are the files formatted correctly?
* [ ] ~Did you add unit tests?~
* [x] Did you check UI tests on the sample app? They are not executed on CI.
* [x] Did you test your change with either the sample apps that are included in the repository or with a blank app that uses your change?

## Description

[AB#83431](https://msmobilecenter.visualstudio.com/454a20a9-dfe6-4c1f-8ae8-417f76adb472/_workitems/edit/83431)

## Related PRs or issues

Enable this warning in the scripts and fix headers.

### CODE REVIEW

File: AppCenter.xcworkspace/xcshareddata/xcschemes/All App Center Frameworks.xcscheme [hunk 1]
```diff
@@ -3,7 +3,7 @@
    LastUpgradeVersion = "1150"
    version = "1.3">
    <BuildAction
-      parallelizeBuildables = "YES"
+      parallelizeBuildables = "NO"
```
Comment (russelarms) [inline AppCenter.xcworkspace/xcshareddata/xcschemes/All App Center Frameworks.xcscheme:6]: I wonder why it doesn't work with parallelized builds.

Comment (AnastasiaKubova) [reply AppCenter.xcworkspace/xcshareddata/xcschemes/All App Center Frameworks.xcscheme:6]: Now the App Center core should assemble everytime when we going to assemble any of our modules and if we run build all our modules in parallel, then the build will be interrupted.

File: AppCenter/AppCenter/Support/iOS.modulemap [hunk 2]
```diff
@@ -4,6 +4,10 @@ framework module AppCenter {
   export *
   module * { export * }
   
+  explicit module Private {
+    header "MSACConstants+Flags.h"
```
Comment (MatkovIvan) [inline AppCenter/AppCenter/Support/iOS.modulemap:8]: Why is it a separate module? Can we just use it like other headers?

File: AppCenter.xcworkspace/xcshareddata/xcschemes/All App Center Frameworks.xcscheme (hunk 1 above)
Comment (MatkovIvan) [reply AppCenter.xcworkspace/xcshareddata/xcschemes/All App Center Frameworks.xcscheme:6]: I'd try to keep fix this problem to keep it parallel

File: CHANGELOG.md [hunk 3]
```diff
@@ -8,6 +8,7 @@
 * **[Breaking change]** The App Center prefix `MS` in Objective-C is consolidated to `MSAC` for collision prevention. Please follow the [migration guide](https://docs.microsoft.com/appcenter/sdk/getting-started/migration/apple-sdk-update) for adopting the new naming.
 * **[Fix]** Fix `NSInvalidArgumentException` when using non-string object as a key in `NSUserDefaults`.
 * **[Fix]** Fix `NSDateFormatter` initialization in a concurrent environment.
+* **[Fix]** Fix `double-quoted` warnings in Xcode 12.
```
Comment (MatkovIvan) [inline CHANGELOG.md:11]: Move it to 4.0.1

**Review by MatkovIvan — CHANGES_REQUESTED** [review]

File: AppCenter/AppCenter/Support/iOS.modulemap (hunk 2 above)
Comment (AnastasiaKubova) [reply AppCenter/AppCenter/Support/iOS.modulemap:8]: No, otherwise he is not works.

File: Sasquatch/SasquatchObjC/AppDelegate.m [hunk 4]
```diff
@@ -18,7 +18,12 @@
 #import "MSACAnalyticsInternal.h"
 
 #elif GCC_PREPROCESSOR_MACRO_SASQUATCH_OBJC
+#if __has_include(<AppCenter/AppCenter.h>)
```
Comment (MatkovIvan) [inline Sasquatch/SasquatchObjC/AppDelegate.m:21]: I guess it's not required here

File: Sasquatch/SasquatchObjC/AppCenterDelegateObjC.mm [hunk 5]
```diff
@@ -13,7 +13,12 @@
 #import "MSACAppCenterInternal.h"
 
 #elif GCC_PREPROCESSOR_MACRO_SASQUATCH_OBJC
+#if __has_include(<AppCenter/AppCenter.h>)
```
Comment (MatkovIvan) [inline Sasquatch/SasquatchObjC/AppCenterDelegateObjC.mm:16]: and here

**Review by MatkovIvan — APPROVED** [review]

**Pull request merged**